### PR TITLE
90 - Linkedin Integration with the UI 

### PR DESCRIPTION
**Pull request by mohanhasti**
JIRA Ticket link: [PC-90](https://makeitmvp.atlassian.net/browse/PC-90)

## Type of change

- [ ] Bug fix
- [x] New feature
- [x] Refactoring

## Description

- Changes made?
integrated the LinkedIn backend to the frontend and made some UI refinements with the ability of pop modal and error/success message alerts 
- Reason for changes?
- Behavior before?
- Behavior after?
- How did you test?

## Checklist

- [x] I have performed a self-review of my code.
- [x] I have manually tested the new feature and found it to be working as expected.
- [x] All commits in this PR are related to the issue.
- [x] Base branch for this PR is set to `develop`.
- [x] JIRA ticket # and my initials are noted in this PR's name (format example: #57-EB Add Pull Request template)

**Reminder**: every PR needs at least 1 peer approval before merging to the `develop` branch. Once a main ticket has been completed, along with all subtasks, associated code can be merged to `main`.

[PC-90]: https://makeitmvp.atlassian.net/browse/PC-90?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mohanhasti** [comment]
JIRA Ticket link: [PC-90](https://makeitmvp.atlassian.net/browse/PC-90)

## Type of change

- [ ] Bug fix
- [x] New feature
- [x] Refactoring

## Description

- Changes made?
Added UI refinements and added go-to LinkedIn functionality
- Reason for changes?
- Behavior before?
- Behavior after?
- How did you test?

## Checklist

- [] I have performed a self-review of my code.
- [x] I have manually tested the new feature and found it to be working as expected.
- [x] All commits in this PR are related to the issue.
- [x] Base branch for this PR is set to `develop`.
- [x] JIRA ticket # and my initials are noted in this PR's name (format example: #57-EB Add Pull Request template)

**Reminder**: every PR needs at least 1 peer approval before merging to the `develop` branch. Once a main ticket has been completed, along with all subtasks, associated code can be merged to `main`.


[PC-90]: https://makeitmvp.atlassian.net/browse/PC-90?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

File: src/components/ErrorMessageAlert/ErrorMessageAlert.jsx
Comment (IguanasEverywhere) [inline src/components/ErrorMessageAlert/ErrorMessageAlert.jsx:5]: Remove if not using?

File: src/components/SuccessMessageAlert/SuccessMessageAlert.scss
Comment (IguanasEverywhere) [inline src/components/SuccessMessageAlert/SuccessMessageAlert.scss]: Throughout CSS, maybe just remove commented out rules that aren't being used

File: src/pages/PromptPage/PromptPage.jsx
Comment (IguanasEverywhere) [inline src/pages/PromptPage/PromptPage.jsx]: I think we can remove this

File: src/pages/PromptPage/PromptPage.jsx
Comment (IguanasEverywhere) [inline src/pages/PromptPage/PromptPage.jsx]: Remove if not using the commented out code?

**Review by IguanasEverywhere — APPROVED** [review]
It seems to be working when I tested it out this morning, thanks for your work on this. This is a huge PR, and given our time frame I think we should go ahead and merge it. In the future, it may be better to do much smaller, more incremental PRs so that the team and mentors can really have to chance to integrate feedback into the process. The merge conflicts appear to be minor, let us know if there are any issues there.

File: src/pages/PromptPage/PromptPage.jsx
Comment (mohanhasti) [reply src/pages/PromptPage/PromptPage.jsx]: Ill go ahead and remove them.

**Comment by mohanhasti** [comment]
Thanks for the review, ill remove the commented unused code and push the code.

**Comment by mohanhasti** [comment]
I have made the changes you suggested. I have pushed in the code. SHould I go ahead and merge it?

**Pull request merged**
